client/state/cancellation-offers/selectors/get-cancellation-offer-apply-success.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Get the success status of applying a cancellation offer
+ *
+ * @param state
+ * @param purchaseId
+ * @returns {any}
```

Comment:
Similar to above, I think we actually know this return type.
```suggestion
 * @returns {boolean}
```

Reply:
thanks! will update 👍